### PR TITLE
adding an ordinary_link macro to summary_table

### PR DESCRIPTION
**Pull request by tommorris**
We sometimes want to be able to have a link in a summary table.

This is needed for a forthcoming PR on @alphagov/digitalmarketplace-supplier-frontend

### CODE REVIEW

File: toolkit/templates/summary-table.html [hunk 1]
```diff
@@ -92,6 +92,12 @@ <h2 class="summary-item-heading">
   {% endcall %}
 {%- endmacro %}
 
+{% macro ordinary_link(service_name, link) -%}
```
Comment (allait) [inline toolkit/templates/summary-table.html]: Why not call it just `link`?

File: toolkit/templates/summary-table.html [hunk 2]
```diff
@@ -92,6 +92,12 @@ <h2 class="summary-item-heading">
...
+  {% call field() %}
+    <a href="{{ link }}">{{ service_name }}</a>
```
Comment (allait) [inline toolkit/templates/summary-table.html]: Is this supposed to be used for `service_name`s only or do we need a better argument name?

**Comment by allait** [comment]
Needs a version bump in `VERSION.txt`

File: toolkit/templates/summary-table.html (hunk 1 above)
Comment (tommorris) [inline toolkit/templates/summary-table.html]: To distinguish it from `service_link`.

File: toolkit/templates/summary-table.html (hunk 2 above)
Comment (tommorris) [inline toolkit/templates/summary-table.html]: Fixed.

**Comment by tommorris** [comment]
Version bump done. Changed it to `link` and removed the reference to "service".

**Comment by robyoung** [comment]
Also needs a rebase, it looks like this branch is few days behind master.

**Comment by tommorris** [comment]
Rebased and stuff.

**Comment by quis** [comment]
Echoing my comment from the supplier frontend PR. This macro should be called `external_link` and styled appropriately.

File: toolkit/templates/summary-table.html [hunk 3]
```diff
@@ -139,6 +145,11 @@ <h2 class="summary-item-heading"{% if id %} id="{{ id }}"{% endif %}>
   {% endcall %}
 {%- endmacro %}
 
+{% macro link(link_title, link) -%}
+  {% call field() %}
+    <a href="{{ link }}">{{ service_name }}</a>
```
Comment (allait) [inline toolkit/templates/summary-table.html]: Still says `service_name`

File: toolkit/templates/summary-table.html [hunk 4]
```diff
@@ -98,6 +98,12 @@ <h2 class="summary-item-heading"{% if id %} id="{{ id }}"{% endif %}>
   {% endcall %}
 {%- endmacro %}
 
+{% macro external_link(service_name, link) -%}
```
Comment (allait) [inline toolkit/templates/summary-table.html]: Should be `link_title`

Comment (tommorris) [inline toolkit/templates/summary-table.html]: Fixed.

File: toolkit/templates/summary-table.html (hunk 3 above)
Comment (tommorris) [inline toolkit/templates/summary-table.html]: Fixed.

**Comment by robyoung** [comment]
Boom, looks good :+1:

File: toolkit/templates/summary-table.html [hunk 5]
```diff
@@ -98,6 +98,12 @@ <h2 class="summary-item-heading"{% if id %} id="{{ id }}"{% endif %}>
   {% endcall %}
 {%- endmacro %}
 
+{% macro external_link(link_title, link) -%}
+  {% call field() %}
+    <a href="{{ link }}" rel="external">{{ link_title }}</a>
```
Comment (allait) [inline toolkit/templates/summary-table.html]: Should we have a `nofollow` here in case it ends up being used by the public pages later?

Comment (quis) [inline toolkit/templates/summary-table.html]: Yes.

**Pull request merged**